src/keystone/keystone_shim.cc

```diff
@@ -32,7 +30,10 @@ bool keystone_getSealingKey(byte* key) {
 bool g_m_initialized = false;
 const int g_m_size = 32;
 byte g_measurement[g_m_size];
-string g_measurement_file_name("./provisioning/keystone_example_app.measurement");
+
+// Keep name consistent with what is being used by other apps.
+// This simplifies the logic in driver run_example.sh script.
+string g_measurement_file_name("./provisioning/example_app.measurement");
```

Comment:
This was a MAJOR bug in the naming stuff. The bash script is expecting the `example_app.measurement`.

Things would not certify as we were provisioning or looking for the wrong measurement file name.